bigchaindb/tep/transactional_election.py

```diff
@@ -0,0 +1,38 @@
+
+class TransactionalElection(object):
+
+    version = '0.0'
```

Comment:
How do we use version and name?

Reply:
As of now it might not be used. But in "future" a class and extend the `UpsertValidator` class and upgrade the version and re-implement necessary methods to achieve desired behavour

Reply:
A new class inherited from `UpsertValidator` will have a different name so they will be distinguishable.

Honestly, I do not see how these fields are useful.

Reply:
Consider a situation wherein a version `2.0` for `UpsertValidator` has to be released because of major changes. In such a case to maintain backward compaitability and validate/replay elections which were proposed using version `1.0` format we would need to have different classes addressing the logic. The current approach tries to store this version information in `version` variable of the class. This hasn't been used anywhere but I hope it would be usefull when need be. Needless to say, if you think it can be addressed in a better way then please let me know.

Reply:
Could you, please, provide more details about how do you think the version can be used? An artificial code sample can help.

Reply:
I imagine it to be used something like this,

```python
class NewUpsertValidator(UpsertValidator):
    version = '2.0'
    def __init__(self):
...    
```

Then at other places in the code,

```python
[for uv in upsert_validator_versions if uv.version == election_txn.asset['data']['version']][0].vote(...) 
```

Reply:
@kansi Maybe you could add more information/explanation about "version" in BEP-20, i.e. https://github.com/bigchaindb/BEPs/pull/45

I noticed that the example there sets "version" to "1.0".

Reply:
Added some details here https://github.com/bigchaindb/BEPs/pull/45/files#diff-8e56cd5a0d784d78d68efd53fa940687R52

Reply:
Regarding versioning and transaction types, I expressed my concerns in the [election BEP](https://github.com/bigchaindb/BEPs/pull/44#issuecomment-393113609).